test_install_firefox fails on macOS arm64

this is obvious enough, but:

```
    def download(self, dest=None, channel="nightly", rename=None):
        product = {
            "nightly": "firefox-nightly-latest-ssl",
            "beta": "firefox-beta-latest-ssl",
            "stable": "firefox-latest-ssl"
        }
    
        os_builds = {
            ("linux", "x86"): "linux",
            ("linux", "x86_64"): "linux64",
            ("win", "x86"): "win",
            ("win", "AMD64"): "win64",
            ("macos", "x86_64"): "osx",
        }
        os_key = (self.platform, uname[4])
    
        if dest is None:
            dest = self._get_browser_binary_dir(None, channel)
    
        if channel not in product:
            raise ValueError("Unrecognised release channel: %s" % channel)
    
        if os_key not in os_builds:
>           raise ValueError("Unsupported platform: %s %s" % os_key)
E           ValueError: Unsupported platform: macos arm64

browser.py:214: ValueError
```

--- a/tools/wpt/browser.py
+++ b/tools/wpt/browser.py
@@ -201,6 +201,7 @@ class Firefox(Browser):
             ("win", "x86"): "win",
             ("win", "AMD64"): "win64",
             ("macos", "x86_64"): "osx",
+            ("macos", "arm64"): "osx",
         }
         os_key = (self.platform, uname[4])
 

--- a/tools/wpt/tests/test_install.py
+++ b/tools/wpt/tests/test_install.py
@@ -57,7 +57,7 @@ def test_install_chrome_chromedriver_by_version():
     if os.path.exists(chromedriver_path):
         os.unlink(chromedriver_path)
     # This is a stable version.
-    binary_path = chrome.install_webdriver_by_version(dest=dest, version="84.0.4147.89")
+    binary_path = chrome.install_webdriver_by_version(dest=dest, version="111.0.5563.146")
     assert binary_path == chromedriver_path
     assert os.path.exists(chromedriver_path)
     os.unlink(chromedriver_path)

